src/parse/rules/python_rules.build_defs

```diff
@@ -125,6 +126,14 @@ def python_binary(name:str, main:str, resources:list=[], out:str=None, deps:list
     cmd = '$TOOLS_PEX -s "%s" -m "%s" --zip_safe' % (shebang, CONFIG.PYTHON_MODULE_DIR)
     pre_build, cmd = _handle_zip_safe(cmd, zip_safe)
 
+    assert main not in srcs, "main file should not be included in srcs"
+    if srcs:
```

Comment:
I'm not sure you need a separate rule here - you should be able to just add them in on line 139 I think?

Reply:
That's what I'd initially tried, but I got a strange error message that led me to suspect something was having difficulty determining where to find `main`. Seemed easier to do things this way than to figure out how pex's actually work ;)

Reply:
Let me know whether you'd like me to try to get this working without an additional target, or if you think this is fine as-is.

Reply:
Ah, I figured out the issue - I had added the `srcs` to `pex_rule` and `lib_rule`, but it was only needed for `lib_rule`  :-) Fixed now!